src/elements/common/modal.scss

```diff
@@ -68,6 +64,4 @@
 
 .be-modal-dialog-overlay {
     @extend .be-modal-wrapper-overlay;
-
-    position: absolute;
 }
```

Comment:
this might cause issues for modals that haven't been converted to blueprint. the styles were added for the previous modal implementation

this block exists to overwrite the base styles for modals that now use blueprint: https://github.com/box/box-ui-elements/blob/86c11327ceb9a29cf49d1fcc83dcb6784db6d54d/src/elements/common/modal.scss#L39-L42